<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReadingForm</class>
 <widget class="QWidget" name="ReadingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>1020</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Манга</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>625</width>
     <height>902</height>
    </rect>
   </property>
   <property name="text">
    <string>Ошибка загрузки страницы</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prevButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>960</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>525</x>
     <y>960</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Вперед</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>327</x>
     <y>960</y>
     <width>23</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>373</x>
     <y>960</y>
     <width>23</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>960</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Выйти в меню</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Автор:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Издательство:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
